.clang-format

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
Documentation on all the options can be found here: https://releases.llvm.org/8.0.0/tools/clang/docs/ClangFormatStyleOptions.html